app/Helpers/helpers.php

```diff
@@ -67,3 +69,32 @@ function htmldir()
         }
     }
 }
+
+if (! function_exists('subclasses')) {
+    /**
+     * Get all subclass of the given class name.
+     *
+     * @template T of object
+     *
+     * @param  class-string<T>  $className
+     * @return \Generator<array-key,ReflectionClass<T>>
+     */
+    function subclasses(string $className): Generator
```

Comment:
If we want to "chipote", `subclasses` should be named `subClasses`

Reply:
done